test/integration/puppetlabs/services/jruby/jruby_pool_int_test.clj

```diff
@@ -296,16 +328,27 @@
                           (start [this context] (lc-fn context :start-bonus-service))
                           (stop [this context] (lc-fn context :stop-bonus-service))
                           (bonus-service-fn [this] (lc-fn nil :bonus-service-fn)))]
+      (fs/delete debug-log)
       (bootstrap/with-puppetserver-running-with-services
        app
        (conj (tk-bootstrap/parse-bootstrap-config! bootstrap/dev-bootstrap-file) bonus-service)
-       {:jruby-puppet {:max-active-instances 1}}
+       {:global {:logging-config
```

Comment:
We're speculating that the interesting data that we'd need to see to figure out why the `@call-seq` assertion fails is buried in some of the debug log output in Trapperkeeper but we can't get at that data by default since we normally suppress debug logging for test runs in order to obey the almighty Wolfe Principle™.  The code here overrides the default logback config with one that cranks up the log level to debug but with an appender that dumps everything into a file.  When the `@call-seq` assertion fails, the full contents of the log file are dumped in the assertion message.  This should hopefully keep us in good graces per the Principle but give us a lot more data to look at when the assertion fires.  Hopefully this can be undone later when we figure out the root cause for why this test has periodically been failing.